lib/twitch-soundboard/static/css/soundboard.css

```diff
@@ -1,4 +1,92 @@
+.divider1 {
```

Comment:
Do we need the '1' here? It doesn't look like there are other dividers or titles and I can't see why we'd need more than one instance of future dividers or titles.